src/test/java/com/epam/test_generator/api/BddGeneratorApi.java

```diff
@@ -0,0 +1,464 @@
+package com.epam.test_generator.api;
```

Comment:
Is it supposed to be here?

Reply:
no, it is always added without my permission, I guessed I deleted it(

Reply:
deleted from commit